sql/status_effects.sql

```diff
@@ -314,7 +314,7 @@ INSERT INTO `status_effects` VALUES (281,'enthunder_ii',41,94,51,0,0,0,5,0,0);
 INSERT INTO `status_effects` VALUES (282,'enwater_ii',41,94,51,0,0,0,6,0,0);
 INSERT INTO `status_effects` VALUES (283,'perfect_defense',32,0,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (284,'egg',32,0,0,0,0,0,0,0,0);
-INSERT INTO `status_effects` VALUES (285,'visitant',32,0,0,0,0,0,0,0,0);
+INSERT INTO `status_effects` VALUES (285,'visitant',33554432,0,0,0,0,0,0,0,0);
```

Comment:
This should be 33554688 to include loss on being zoned out.

Reply:
Ok I will try that out, this is non-removable by the user, lost upon zoning out, and is removed via time out I will check with that number.

Reply:
src\status_effects.h has the flags enumerated in hex. Can add them up and convert to decimal in windows windows calculator (or similar) to see the combined value for here.

Reply:
I had no idea "being clicked off via add-ons" was a thing